Library/Homebrew/tap.rb

```diff
@@ -25,12 +27,15 @@ class Tap
   # e.g. `/usr/local/Library/Taps/user/homebrew-repo`
   attr_reader :path
 
+  attr_reader :json_path
+
   def initialize(user, repo)
     # we special case homebrew so users don't have to shift in a terminal
     @user = user == "homebrew" ? "Homebrew" : user
     @repo = repo
     @name = "#{@user}/#{@repo}".downcase
     @path = TAP_DIRECTORY/"#{@user}/homebrew-#{@repo}".downcase
+    @json_path = TAP_DIRECTORY/"#{@user}/#{@repo}.json".downcase
```

Comment:
My concern is that this may not be ignored by `.gitignore` in the taps. Perhaps this file should be installed into `Library/Taps/.#{@{user}_homebrew-#{@repo}` or something?